### ISSUE TITLE
Strip sidebar `a[title]` attributes

### ISSUE THREAD

**Issue by peterbe** [mdn/yari#1248]
It was a hard battle but after much research, debating, and some hacking we got rid of the `title` tooltips on all the links in the sidebars in Yari (we also did the xrefs by the way).
Not only are they bad for the a11y but they're also adding an enormous amount of bytes to the rendered HTML.
I wrote a Python script that looked at deleting all `#Quick_Links a[title]` attributes and then you get this:

TOTAL_SIZE_BEFORE 252.6MB
TOTAL_SIZE_AFTER 199.0MB
TOTAL_SIZE_DIFFERENCE 53.6MB
TOTAL_NUMBER_OF_DOCS 16682

Basically, if we apply this manipulation, we stand to make the archive content ~25% smaller. That's less electricity on S3 and CDN and users having to download something they never want. All we need to do is one simple cheerio command.

**Comment by peterbe** [comment]
Oops. Forgot to close this issue with https://github.com/mdn/yari/pull/1258
